castervoice/apps/dragon.py

```diff
@@ -18,11 +18,42 @@ def fix_dragon_double(nexus):
     except Exception:
         utilities.simple_log(False)
 
+def cap_dictation(dictation):
+    input_list = str(dictation).split(" ")
+    output_list = []
+    for i in range(len(input_list)):
+        if input_list[i] == "cap":
+            input_list[i+1] = input_list[i+1].title()
+        else:
+            output_list.append(input_list[i])
+    Text(" ".join(output_list)).execute()
+
+# extras are common to both classes in this file
+extras_for_whole_file = [
+        Dictation("text"),
```

Comment:
Do you use all of these Dictation objects? Can you change it so that "text" is usually used to be consistent with `nav.py`?